tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -205,10 +325,22 @@ def auto(net: TensorNetwork,
   return greedy(net, output_edge_order, memory_limit)
 
 
-def custom(net: TensorNetwork,
-           optimizer: Any,
-           output_edge_order: Sequence[Edge] = None,
-           memory_limit: Optional[int] = None) -> TensorNetwork:
+def auto(
+    nodes: Union[TensorNetwork, Iterable[BaseNode]],
+    output_edge_order: Sequence[Edge] = None,
+    memory_limit: Optional[int] = None):  # -> Union[BaseNode, TensorNetwork]:
+
+  if isinstance(nodes, TensorNetwork):
+    return _auto_network(nodes, output_edge_order, memory_limit)
```

Comment:
Would it not just be possible to do `_auto_nodes(net.nodes_set,...)` instead of having 2 different methods?

Reply:
fixed